tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -0,0 +1,148 @@
+#Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Training loop for PointNet v1 on modelnet40."""
+# pylint: disable=missing-function-docstring
+
+import tensorflow as tf
+from tqdm import tqdm
+import helpers
+import augment
+from tensorflow_graphics.datasets.modelnet40 import ModelNet40
```

Comment:
sort imports in alphabetical order.

Reply:
can we enable something in pylint for this? done